contracts/fuelchain/FuelChainConsensus.sol

```diff
@@ -7,21 +8,42 @@ import {AccessControlUpgradeable} from "@openzeppelin/contracts-upgradeable/acce
 import {PausableUpgradeable} from "@openzeppelin/contracts-upgradeable/security/PausableUpgradeable.sol";
 import {CryptographyLib} from "../lib/Cryptography.sol";
 
+/// @notice Structure for epoch commits
+struct Commit {
+    bytes32 blockHash;
+    uint32 timestamp;
+    address reserved1;
+    uint16 reserved2;
+}
+
 /// @notice The Fuel v2 chain state consensus
 contract FuelChainConsensus is Initializable, PausableUpgradeable, AccessControlUpgradeable, UUPSUpgradeable {
     ///////////////
     // Constants //
     ///////////////
 
+    /// @dev The commit proccess parameters
+    //TODO: update these values once block time and epoch size are finalized
+    uint256 public constant NUM_COMMIT_SLOTS = 240; //30 days worth of commits
+    uint256 public constant BLOCKS_PER_EPOCH = 10800;
+    uint256 public constant TIME_TO_FINALIZE = 10800;
+
     /// @dev The admin related contract roles
     bytes32 public constant PAUSER_ROLE = keccak256("PAUSER_ROLE");
+    bytes32 public constant COMMITTER_ROLE = keccak256("COMMITTER_ROLE");
+
+    ////////////
+    // Events //
+    ////////////
+
+    event CommitSubmitted(uint256 indexed epoch, bytes32 blockHash);
```

Comment:
"CommitSubmitted" almost seems redundent? 
This could maybe be changed to just "Committed" or "ComitMade"
minor though, not critical.

Reply:
This will be addressed in #21